chore: add v5 migration guide (#67)
terraform-google-modules/terraform-google-log-export: docs/upgrading_to_v5.0.md

# Upgrading to Log Export v5.0

The v5.0 release of Log Export is a backwards incompatible release and features many quality of life improvements.

> Changes are only made to the storage and bigquery submodules.

Reviewer: ```suggestion
Breaking changes have only been made to the storage and bigquery submodules.
```

Other modules can safely update the version without needing any changes.

## Migration Instructions

> NOTE: Users should prefer to let terraform update their resources to the newer defaults.

Reviewer: ```suggestion
NOTE: Users should prefer to let Terraform update their resources to the newer defaults.
```

To preserve the existing defaults, see below:

```diff

> module "gcs" {

Reviewer: nit:
```suggestion
module "bq" {
```
